index.js

```diff
@@ -2,128 +2,14 @@
-      filepath = paths[i];
-      if (!CSS_SUFFIX.test(filepath)) { continue; }
-      var podPath = filepath.split('/').slice(0, -1);
-      var podGuid = podPath.join('--') + '-' + guid();
-      var cssFileContents = fs.readFileSync(path.join(srcDir, filepath)).toString();
-      var parsedCss = css.parse(cssFileContents);
-      var transformedParsedCSS = transformCSS(podGuid, parsedCss);
-      buffer.push(css.stringify(transformedParsedCSS));
-      podLookup[podPath.join('/')] = podGuid;
-    }
-
-    fs.writeFileSync(path.join(destDir, 'pod-styles.css'), buffer.join(''));
-    fs.writeFileSync(path.join(destDir, 'pod-lookup.json'), JSON.stringify(podLookup));
-  });
-};
-
-function ComponentCssPostprocessor(inputTree) {
-  this.inputTree = inputTree;
-}
-
-ComponentCssPostprocessor.prototype = Object.create(Writer.prototype);
-ComponentCssPostprocessor.prototype.constructor = ComponentCssPostprocessor;
-
-ComponentCssPostprocessor.prototype.write = function (readTree, destDir) {
-  return readTree(this.inputTree).then(function(srcDir) {
-    var paths = walkSync(srcDir);
-    var currentPath;
-    var cssInjectionSource;
-    for (var i = 0, l = paths.length; i < l; i++) {
-      currentPath = paths[i];
-      if (currentPath === "pod-lookup.json") {
-        var podLookupFilepath = path.join(srcDir, "pod-lookup.json");
-        var podLookup = fs.readFileSync(podLookupFilepath);
-        cssInjectionSource = "\n\nEmber.COMPONENT_CSS_LOOKUP = " + podLookup + ";\n";
-        cssInjectionSource += "Ember.ComponentLookup.reopen({\n" +
-          "  lookupFactory: function(name, container) {\n" +
-          "    var Component = this._super(name, container);\n" +
-          "    if (!Component) { return; }\n" +
-          "    return Component.reopen({\n" +
-          "      classNames: [Ember.COMPONENT_CSS_LOOKUP[name]]\n" +
-          "    });\n" +
-          "  }\n" +
-          "});\n";
-      } else {
-        if (currentPath[currentPath.length-1] === '/') {
-          fs.mkdirSync(path.join(destDir, currentPath));
-        } else {
-          symlinkOrCopy.sync(path.join(srcDir, currentPath), path.join(destDir, currentPath));
-        }
-      }
-    }
-
-    fs.appendFileSync(path.join(destDir, "assets", "vendor.js"), cssInjectionSource);
-    fs.appendFileSync(path.join(destDir, "assets", "vendor.css"), fs.readFileSync(path.join(srcDir, 'pod-styles.css')));
-  });
-};
-
-function ComponentCSSPreprocessor(options) {
-  this.name = 'component-css';
-  this.options = options || {};
-}
-
-ComponentCSSPreprocessor.prototype.toTree = function(tree, inputPath, outputPath) {
-  var filteredTree = new Funnel(tree, {
-    srcDir: this.options.podDir || 'app',
-    exclude: [/^styles/]
-  });
-  return new BrocComponentCssPreprocessor(filteredTree);
-};
+var ComponentCssPreprocessor = require('./ComponentCssPreprocessor');
+var ComponentCssPostprocessor = require('./ComponentCssPostprocessor');
```

Comment:
Can and will do. This was quick, so I didn't put much thought into file organization.